e2e/utils/element-utils.ts

```diff
@@ -67,3 +67,11 @@ export async function matchText(page: Page, cssSelector: string, subString: stri
   }
   return found;
 }
+
+export async function exists(page: Page, xpath: string, opt: { timeout?: number } = {}): Promise<boolean> {
```

Comment:
Perhaps we should rename this to `isVisible`; something that matches the semantics of what the underlying Puppeteer check is doing. If we want to use `exists` as the name, then probably we should have `visible` be an option - which could default to true.

Reply:
I like the second suggestion. Sometimes we may want to check if an element exists but not visible.
Updated func to
```
export async function exists(
  page: Page,
  xpath: string,
  opt: { visible?: boolean; timeout?: number } = {}
): Promise<boolean> {
  const { visible = true, timeout = 1000 } = opt;
  return page
    .waitForXPath(xpath, { visible, timeout })
    .then(() => true)
    .catch(() => false);
}
```